model/pytorch/dffml_model_pytorch/utils/utils.py

```diff
@@ -34,6 +33,45 @@ def create_layer(layer_dict):
     return sequential_dict
 
 
+def create_network(network_dict):
+    sequential_network = nn.Sequential()
+    for name, layer in network_dict.items():
+        if isinstance(layer, dict):
+            parameters = {k: v for k, v in layer.items()}
+            layer_type = parameters.pop("layer_type")
+            try:
+                if not isinstance(parameters["in_channels"], int):
```

Comment:
`in_channels ` may or may not be present so we are handling it in `try` and `except` (line 53) block , In except we need not do any thing as `in_channels` may be absent like `ReLU, MaxPool ....`

Reply:
I think the if else statements can handle that. We shouldn't need try except blocks for that.

Reply:
okay I will change it